Package/gulpfile.js

```diff
@@ -301,6 +304,32 @@ const copyVCXProjUWPFiles = () => {
     .pipe(gulp.dest('Assembled-Windows/windows/BabylonReactNative'));
 }
 
+const copyOpenXRInfoFiles = () => {
+  return gulp.src('../Modules/@babylonjs/react-native/submodules/BabylonNative/Dependencies/xr/Dependencies/OpenXR-MixedReality/LICENSE')
+  .pipe(gulp.src('../Modules/@babylonjs/react-native/submodules/BabylonNative/Dependencies/xr/Dependencies/OpenXR-MixedReality/README.md'))
+  .pipe(gulp.dest('Assembled-Windows/windows/OpenXR-MixedReality'));
+}
+
+const copyOpenXRPreviewHeaders = () => {
```

Comment:
Does it really make sense to have these as four separate functions, or should we just combine them into a single copyOpenXRHeaders that internally uses gulp.parallel?

Reply:
i have separate functions for files that copy to different directories. I'll go a head and combine copyOpenXRExtensionHeader and copyOpenXRHelperHeader